draft-ietf-quic-recovery.md

```diff
@@ -427,11 +427,10 @@ Pseudocode for DetectLostPackets follows:
    DetectLostPackets(acked_packet):
      lost_packets = {};
      foreach (unacked_packet less than acked_packet):
-       if (unacked_packet.time_sent <
-           acked_packet.time_sent - kTimeReorderThreshold * smoothed_rtt):
+       delta = acked_packet.time_sent - unacked_packet.time_sent
+       if (delta > kTimeReorderThreshold * smoothed_rtt):
          lost_packets.insert(unacked_packet.packet_number);
-       else if (unacked_packet.packet_number <
-                acked_packet.packet_number - reordering_threshold)
+       else if (delta > reordering_threshold)
```

Comment:
Ugh, I realized this yesterday, fixed it, but failed to push the update.  On it.